docker-bake.hcl

```diff
@@ -32,6 +32,20 @@ target "release" {
   output = [DOCS_SITE_DIR]
 }
 
+target "image" {
+  args = {
+    HUGO_ENV = HUGO_ENV
+    DOCS_URL = "/"
+  }
+  target = "image"
+  attest = [
+    "type=provenance,mode=max",
+    "type=sbom",
+  ]
+  tags = ["docs/docker-docs:latest"]
```

Comment:
Maybe just `docker-docs` as tag is enough?

```suggestion
  tags = ["docker-docs"]
```

Reply:
I thought we would push this to the registry under the `docs` free team https://hub.docker.com/u/docs

Reply:
unset the tags attr in the bake file and set it in the gha workflow instead